README.md

```diff
@@ -53,7 +53,7 @@ If you have a comment or suggestion, please open an [Issue](https://github.com/d
 - [Rotating keys](#rotating-keys)
   - [Setup environment](#setup-environment)
   - [Renewing sub-keys](#renewing-sub-keys)
-  - [Rotating keys](#rotating-keys)
+  - [Rotating keys](#rotating-keys-1)
```

Comment:
I had to click this like a dozen times last night; it goes to the h1 instead of the h2 without the numeric suffix!